### ISSUE TITLE
Tracebacks not logged

### ISSUE THREAD

**Issue by walles** [richq/folders2flickr#44]
I'm getting some exceptions while running, but the tracebacks aren't logged.

Could you please make sure the tracebacks get logged so that I can send in useful problem reports?

  Regards /Johan

Sample output with tracebackless exceptions:
```
2015-05-01 20:19:59,469 uploadr.py:498 -          uploadImage() Uploading image C:\Users\Acer\Pictures\2013-11-29\DSCN6370-001.JPG with tags "#\2013-11-29\DSCN6370-001.JPG" "" "2013-11-29"
2015-05-01 20:20:15,621 uploadr.py:498 -          uploadImage() Uploading image C:\Users\Acer\Pictures\2013-11-29\DSCN6370.JPG with tags "#\2013-11-29\DSCN6370.JPG" "" "2013-11-29"
2015-05-01 20:20:28,536 uploadr.py:498 -          uploadImage() Uploading image C:\Users\Acer\Pictures\2013-11-29\DSCN6371-001.AVI with tags "#\2013-11-29\DSCN6371-001.AVI" "" "2013-11-29"
2015-05-01 20:20:31,851 uploadr.py:532 -          uploadImage() (<type 'exceptions.MemoryError'>, MemoryError(), <traceback object at 0x02A58738>)
2015-05-01 20:20:31,944 uploadr.py:498 -          uploadImage() Uploading image C:\Users\Acer\Pictures\2013-11-29\DSCN6371.AVI with tags "#\2013-11-29\DSCN6371.AVI" "" "2013-11-29"
2015-05-01 20:20:35,183 uploadr.py:532 -          uploadImage() (<type 'exceptions.MemoryError'>, MemoryError(), <traceback object at 0x02A589B8>)
2015-05-01 20:20:35,653 uploadr.py:498 -          uploadImage() Uploading image C:\Users\Acer\Pictures\2013-11-29\DSCN6372-001.JPG with tags "#\2013-11-29\DSCN6372-001.JPG" "" "2013-11-29"
2015-05-01 20:20:51,812 uploadr.py:498 -          uploadImage() Uploading image C:\Users\Acer\Pictures\2013-11-29\DSCN6372.JPG with tags "#\2013-11-29\DSCN6372.JPG" "" "2013-11-29"
2015-05-01 20:21:07,964 uploadr.py:498 -          uploadImage() Uploading image C:\Users\Acer\Pictures\2013-11-29\DSCN6373-001.JPG with tags "#\2013-11-29\DSCN6373-001.JPG" "" "2013-11-29"
```

**Comment by richq** [comment]
This looks like issues #40 and #21, which I have been totally unable to fix as it is a bug in the version of exif-py that is compatible with python2. Exif-py has moved to python3 so that's my goal now. The problem is in parsing the metadata in the photos, there's something that goes wrong and it tries to read in about a squillion bytes instead of the camera name (or whatever).

**Comment by walles** [comment]
Merging pull request #45 would fix this.

  Regards /Johan

**Comment by walles** [comment]
Fixed by pull request #45.
